addons/web/static/src/js/fields/relational_fields.js

```diff
@@ -2756,6 +2756,13 @@ var FieldSelection = AbstractField.extend({
         var required = this.attrs.modifiersValue && this.attrs.modifiersValue.required;
         for (var i = 0 ; i < this.values.length ; i++) {
             var disabled = required && this.values[i][0] === false;
+            console.log(this.values);
+            console.log(i);
+            console.log(this.values[i])
+            // console.log(this.values[i][0])
+            if (this.values[i] == undefined) {
+                debugger
```

Comment:
!?

Reply:
my ho my ... >_<

Reply:
Oopsie, this part can obviously be reverted...
I'll do it right away.

However, there is a bug somewhere in those lines. To reproduce it, you just have to put a `widget='selection'` on a Many2one field located in a One2many field in a form view.